dask_snowflake/tests/test_core.py

```diff
@@ -52,7 +53,7 @@ def test_write_read_roundtrip(table, connection_kwargs, client):
     to_snowflake(ddf, name=table, connection_kwargs=connection_kwargs)
 
     query = f"SELECT * FROM {table}"
-    df_out = read_snowflake(query, connection_kwargs=connection_kwargs)
+    df_out = read_snowflake(query, connection_kwargs=connection_kwargs, npartitions=2)
```

Comment:
Maybe throw the npartitions=2 into `connection_kwargs` ?  Maybe a bad idea

Reply:
I don't love this idea, I want `connection_kwargs` to be exactly what needs to be passed to the snowflake python connector, and not have any dask-related logic

Reply:
Ah sorry, I thought that connection_kwargs was a parameter that was splatted out.  I agree with not changing the actual code.  I was looking for a sneaky way to affect all of the tests.